examples/pollbot.py

```diff
@@ -0,0 +1,147 @@
+        context.bot.stop_poll(context.bot_data[poll_id]["chat_id"],
+                              context.bot_data[poll_id]["message_id"])
+
+
+def quiz(update, context):
+    """Send a predefined poll"""
+    questions = ["1", "2", "4", "20"]
+    message = update.effective_message.reply_poll("How many eggs do you need for a cake?",
+                                                  questions, type=Poll.QUIZ, correct_option_id=2)
+    # Save some info about the poll the bot_data for later use in receive_quiz_answer
+    payload = {message.poll.id: {"chat_id": update.effective_chat.id,
+                                 "message_id": message.message_id}}
+    context.bot_data.update(payload)
+
+
+def receive_quiz_answer(update, context):
+    """Close quiz after three participants took it"""
+    # the bot can receive closed poll updates we don't care about
+    if update.poll.is_closed:
+        return
+    if update.poll.total_voter_count == 3:
+        try:
+            quiz_data = context.bot_data[update.poll.id]
+        # this means this poll answer update is from an old poll, we can't stop it then
+        except KeyError:
+            return
+        context.bot.stop_poll(quiz_data["chat_id"], quiz_data["message_id"])
+
+
+def preview(update, context):
+    """Ask user to create a poll and display a preview of it"""
+    # using this without a type lets the user chooses what he wants (quiz or poll)
+    button = [[KeyboardButton("Press me!", request_poll=KeyboardButtonPollType())]]
+    message = "Press the button to let the bot generate a preview for your poll"
+    # using one_time_keyboard to hide the keyboard
+    update.effective_message.reply_text(message,
+                                        reply_markup=ReplyKeyboardMarkup(button,
+                                                                         one_time_keyboard=True))
+
+
+def receive_poll(update, context):
+    """On receiving polls, reply to it by a closed poll copying the received poll"""
+    actual_poll = update.effective_message.poll
+    # Only need to set the question and options, since all other parameters don't matter for
+    # a closed poll
+    update.effective_message.reply_poll(
+        question=actual_poll.question,
+        options=[o.text for o in actual_poll.options],
+        # with is_closed true, the poll/quiz is immediately closed
+        is_closed=True,
+        reply_markup=ReplyKeyboardRemove()
+    )
+
+
+def help_handler(update, context):
+    """Display a help message"""
+    update.message.reply_text("Use /quiz, /poll or /preview to test this "
+                              "bot.")
+
+
+def main():
+    # Create the Updater and pass it your bot's token.
+    # Make sure to set use_context=True to use the new context based callbacks
+    # Post version 12 this will no longer be necessary
+    updater = Updater("TOKEN", use_context=True)
+    dp = updater.dispatcher
+    dp.add_handler(CommandHandler('start', start))
```

Comment:
Is there any way you could add an example for a ConversationHandler? I wanted to test out using 3 polls back to back aka when one finishes, the second pops up. When the 3rd finishes, I would use the payload data to perform some sort of action and send a message. If this is too much to ask, then please ignore my comment.

Thank you!

Reply:
Just to clarify: We already have examples for `ConversationHandler` [here](https://github.com/python-telegram-bot/python-telegram-bot/tree/master/examples).
If you already saw them and it's about adding an example for CH with polls: Personally, I think that's a bit too specific for the examples. If you have a question on how to set your CH up, you're still welcome to ask in the [Telegram Group](https://t.me/pythontelegrambotgroup) or open an issue using the question template ;)

Reply:
```python
    start_handler = ConversationHandler(
        entry_points=[CommandHandler('poll', poll)],
        states={
            1: [PollAnswerHandler(receive_poll_answer), # Tried with an without this MessageHandler(Filters.poll, receive_poll),
]
        },
        fallbacks=[CommandHandler('poll', poll)],
    )

def poll():
    # Same as before, but returns 1
    return 1
```

I agree it is specific, but the reason I am asking here is because this MR hasn't been merged yet, soon hopefully. I ran your example and it works perfectly. Just with ConversationHandler it doesn't. I've used ConversationHandler before and have got it working.

If the question template, allows for question related to pending MRs, I can do that

Reply:
Yes, please use the question template. Comments on PRs are for discussing the code changes only. Please make sure to fill out the entire template and maybe try to provide a minimal (not) working example.